src/package.rs

```diff
@@ -84,3 +84,13 @@ pub fn prepare_package(package_url: String, package_archive: String, output_dire
     }
     Ok(())
 }
+
+pub fn remove_package(package_archive: String, output_directory: String) -> Result<()> {
+    if Path::new(&package_archive).exists() {
+        fs::remove_file(&package_archive);
+    }
+    if Path::new(&output_directory).exists() {
+        fs::remove_dir_all(output_directory).unwrap();

```

Comment:
Won't be better to propagate error instead of panic?
```suggestion
        fs::remove_dir_all(output_directory)?

```

Reply:
Thank you @kumekay . This place can lead to error, e.g. when the Companion process is still running which will block delete operation on Windows. 

Please review updated code with `anyhow`.